_dashboards/get-started/dql.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Dashboards Query Language
+parent: OpenSearch Dashboards core concepts
+nav_order: 40
+redirect_from:
+  - /dashboards/dql/
+---
+
+# Dashboards Query Language
+
+Dashboards Query Language (DQL) is a simple text-based query language for filtering data in OpenSearch Dashboards. Similar to [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/index), DQL uses HTTP request body. For example, to display your site visitor data for a host in the United States, you would enter `geo.dest:US` into the search field.
+
+This documentation uses the Dashboards web log sample data set. See [Quickstart for OpenSearch Dashboards]() to learn how to add sample data to Dashboards.
+{: .note}
+
+## Terms query
+
+The most basic query is to specify the search term.
+
+```
+host:www.example.com
+```
+
+To access an object's nested field, list the complete path to the field separated by periods. For example, to retrieve the `lat` field in the `coordinates` object:
```

Comment:
The end of the sentence could use some clarification. Suggestion: "For example, to retrieve the `lat` field in the `coordinates` object, you can list the following path:"